src/components/vm/hostdevs/hostDevCard.jsx

```diff
@@ -97,40 +95,40 @@ export const VmHostDevCard = ({ vm, nodeDevices, config }) => {
             slot = String(slot).padStart(2, '0');
             func = String(func).padStart(1, '0');
 
-            addOptionalToCell(cell, `${domain}:${bus}:${slot}.${func}`, `${hostdevId}-slot`, _("Slot"));
+            cells.push(getOptionalValue(`${domain}:${bus}:${slot}.${func}`, `${hostdevId}-slot`, _("Slot")));
         } else if (hostDev.type === "scsi") {
             const bus = hostDev.source.address.bus;
             const target = hostDev.source.address.target;
             const unit = hostDev.source.address.lun;
 
-            addOptionalToCell(cell, bus, `${hostdevId}-bus`, _("Bus"));
-            addOptionalToCell(cell, unit, `${hostdevId}-unit`, _("Slot"));
-            addOptionalToCell(cell, target, `${hostdevId}-target`, _("Target"));
+            cells.push(getOptionalValue(bus, `${hostdevId}-bus`, _("Bus")));
+            cells.push(getOptionalValue(unit, `${hostdevId}-unit`, _("Slot")));
+            cells.push(getOptionalValue(target, `${hostdevId}-target`, _("Target")));
         } else if (hostDev.type === "scsi_host") {
             const protocol = hostDev.source.protocol;
             const wwpn = hostDev.source.wwpn;
 
-            addOptionalToCell(cell, protocol, `${hostdevId}-protocol`, _("Protocol"));
-            addOptionalToCell(cell, wwpn, `${hostdevId}-wwpn`, _("WWPN"));
+            cells.push(getOptionalValue(protocol, `${hostdevId}-protocol`, _("Protocol")));
+            cells.push(getOptionalValue(wwpn, `${hostdevId}-wwpn`, _("WWPN")));
         } else if (hostDev.type === "mdev") {
             const uuid = hostDev.source.address.uuid;
 
-            addOptionalToCell(cell, uuid, `${hostdevId}-uuid`, _("UUID"));
+            cells.push(getOptionalValue(uuid, `${hostdevId}-uuid`, _("UUID")));
         } else if (hostDev.type === "storage") {
             const block = hostDev.source.block;
 
-            addOptionalToCell(cell, block, `${hostdevId}-block`, _("Block"));
+            cells.push(getOptionalValue(block, `${hostdevId}-block`, _("Block")));
         } else if (hostDev.type === "misc") {
             const ch = hostDev.source.char;
 
-            addOptionalToCell(cell, ch, `${hostdevId}-char`, _("Char"));
+            cells.push(getOptionalValue(ch, `${hostdevId}-char`, _("Char")));
```

Comment:
Same as in case above. Libvirt docs specify it's a mandatory field https://libvirt.org/formatdomain.html#block-character-devices, just as libvirt XML schema, so it should always be present.